content/doc/administrate/domain-names.md

```diff
@@ -16,41 +16,42 @@ type: docs
 When creating an application, you have two (non-exclusive) choices for domain names:
 
 * Using a personal domain name
-* Using a `cleverapps.io` free domain for dev purposes, with built-in SSL
+* Using a `cleverapps.io` free domain for development and testing purposes, with built-in SSL
 
 Add it in the application configuration: in the console, click **application name** in the first panel, then choose **domain names**. You'll have to choose to add a custom domain name or use a subdomain under `*.cleverapps.io`.
 
 {{< callout type="info" >}}
   You can link one or more domain names to the same application. On the other hand, you can add `mydomain.com/app1` and `mydomain.com/app2` to different Clever Cloud applications at the same time through [path routing](#path-routing).
 {{< /callout >}}
 
-## Using a **\*.cleverapps.io** free domain, with built-in SSL
+## Testing with `cleverapps.io` Domain
 
 {{< callout type="warning" >}}
-  `*.cleverapps.io` domains are given for development and tests purpose. They point to
-  specific reverse proxies and have the following weaknesses: `.io` TLD is not a stable one, and since we offer the domain, the probability that people will abuse it is high. Thus, we do not guarantee the same QoS on the `cleverapps` reverse proxies.
+  Clever Cloud provides `*.cleverapps.io` domains for development and testing purposes. They point to specific reverse proxies and have the following drawbacks: the `.io` TLD is not made for production, and since we offer the domain, the likelihood of people abusing it can be very high. **It is therefore not possible to guarantee a high level of quality on cleverapps domains**.
```

Comment:
Nitpicking challenge accepted

```suggestion
  Clever Cloud provides `*.cleverapps.io` domains for development and testing purposes. They point to specific reverse proxies and have the following drawbacks: the `.io` TLD isn't made for production, and since we offer the domain, the likelihood of people abusing it can be very high. **It's therefore not possible to guarantee a high level of quality on cleverapps domains**.
```